### PR TITLE
Fixes #2.

### LINKED ISSUE #2: Polyfill Promise for node version < 0.12
On IOJS and Node v.0.12 a native Promise implementation is already provided.  On versions that do not have a native implementation, then use Bluebird.

[re: node-polyfill-promise](https://github.com/jden/node-polyfill-promise)

### PR DESCRIPTION
**Pull request by donaldjarmstrong**
Replaced bluebird with node-polyfill-promise.
Updated the example to not use a Bluebird specific method.
Tests are passing in node v0.10.38 and iojs v2.2.1

### CODE REVIEW

**Comment by donaldjarmstrong** [comment]
I love this module!  The couple of applications that use this module are migrating to iojs and node v0.12 and am in favor of leveraging the native promise implementation if one exists.

File: README.md [hunk 1]
```diff
@@ -75,10 +74,10 @@ An unofficial client for *reading* data from Google Sheets, since [googleapis do
         sheets.getRange(documentId, sheetInfo.id, 'A1:C3')
       ]);
     })
-    .spread(function(sheet, rows) {
-      // TODO: Do something with cell within range
-      console.log('Sheets contents:', sheet, rows);
-    })
+	.then(function(sheets) {
```
Comment (juhamust) [inline README.md:77]: Please use 2 spaces instead of tabs, otherwise looks good!

Comment (juhamust) [inline README.md:77]: Well, I went ahead and fixed this before the merge.

**Comment by juhamust** [comment]
Excellent! Thanks for the contribution, Donnie!

**Comment by donaldjarmstrong** [comment]
Great!  I've used this lib many times with great results.  Love it!!! Keep up the good work

**Pull request merged**